src/libs/Navigation/Navigation.js

```diff
@@ -194,7 +194,7 @@ function isActiveRoute(routePath) {
  *
  * Example:
  * ```jsx
- * if (!Permissions.canUseFreePlan(this.props.betas)) {
+ * if (!Permissions.canUseDefaultRooms(this.props.betas)) {
```

Comment:
Hm, I'm not sure why a component was ever defined inside a lib (tsk tsk), but it appears like the `DismissModal` component isn't used at all anymore now and can be completely removed?

Reply:
I'll fix the lint errors and remove DismissModal too.